lib/personalized_search/product_preferences_manager.dart

```diff
@@ -102,6 +102,18 @@ class ProductPreferencesManager {
     return result;
   }
 
+  /// Returns whether an attribute is important as per the user preferences.
+  bool isAttributeImportant(String attributeId) {
```

Comment:
Should this return a `bool?`, like the other methods relying on `_availablePreferenceImportances` in this file?

Reply:
It depends upon whether we want to continue having 4 levels of "PreferenceImportance" or turn it into a boolean, my understanding is that we want to turn it into a boolean as of now, but if we later decide to continue having the 4 levels I would  modify the method to getAttributeImportance and return PreferenceImportance, but as of now, I think this is fine

Reply:
This is just a question of consistency of the API. Take a look at `getImportanceIndex()`, for example. It returns `null` if and only if `_availablePreferenceImportances` is `null`. I don't really know what that means, but I think it would be more consistent for `isAttributeImportant()` to implement the same behaviour.

Reply:
I see what you mean, I mistook your original comment to be "Should this return a bool?" :D 

you're right, I'll send a follow up PR.